itou/approvals/models.py

```diff
@@ -1214,6 +1214,28 @@ def deny(self, user):
         notifications.ProlongationRequestDeniedJobSeeker(self).send()
 
 
+class ProlongationRequestDenyInformation(models.Model):
+    prolongation_request = models.OneToOneField(
```

Comment:
Belle implé, mais le "1-to-1" s'imposait vraiment ?
L'arrayField est "à plat" et le reste des champs n'est pas vraiment lourd et au final qualifient la demande de prolongation elle-même.

Reply:
Pareil, un peu partagé (mais sans avis ferme :stuck_out_tongue: ).
L'avantage d'un seul modèle c'est qu'on peut imaginer des contraintes vérifiant que les champs deny_xxx ne sont remplis que si le `status` est `DENIED` (mais c'est un peu faible :grin: )